shub/utils.py

```diff
@@ -476,6 +472,16 @@ def latest_github_release(force_update=False, timeout=1., cache=None):
     return release_data
 
 
+def read_release_data(cache):
```

Comment:
Hm I don't really see an advantage in outsourcing this function but I also have no strong opinion on it. ;) If you do outsource it should be generic though, e.g. `def read_json_dict(filename):`